ci/setup_ndk.sh

```diff
@@ -26,4 +25,11 @@ ln -s "$TOOLCHAIN/bin/aarch64-linux-android21-clang"    "$TOOLCHAIN/bin/aarch64-
 ln -s "$TOOLCHAIN/bin/armv7a-linux-androideabi21-clang" "$TOOLCHAIN/bin/arm-linux-androideabi-gcc"
 ln -s "$TOOLCHAIN/bin/armv7a-linux-androideabi21-clang" "$TOOLCHAIN/bin/arm7a-linux-androideabi-gcc"
 
-echo "$ANDROIDNDK/android-ndk-r21e/toolchains/llvm/prebuilt/linux-x86_64/bin" >> $GITHUB_PATH
+
+ln -s "$TOOLCHAIN/bin/llvm-ar" "$TOOLCHAIN/bin/x86_64-linux-android-ar"
+ln -s "$TOOLCHAIN/bin/llvm-ar" "$TOOLCHAIN/bin/i686-linux-android-ar"
+ln -s "$TOOLCHAIN/bin/llvm-ar" "$TOOLCHAIN/bin/aarch64-linux-android-ar"
+ln -s "$TOOLCHAIN/bin/llvm-ar" "$TOOLCHAIN/bin/arm-linux-androideabi-ar"
+ln -s "$TOOLCHAIN/bin/llvm-ar" "$TOOLCHAIN/bin/arm7a-linux-androideabi-ar"
+
+echo "$ANDROIDNDK/android-ndk-r26/toolchains/llvm/prebuilt/linux-x86_64/bin" >> $GITHUB_PATH
```

Comment:
why now there's the need of these symlinks?

Reply:
I guess because the removal of `strip = f"{TOOLCHAIN}/{NDK_MAP[config.arch]}/bin/strip"`?